cms/envs/production.py

```diff
@@ -357,11 +357,12 @@
 AWS_BUCKET_ACL = AWS_DEFAULT_ACL
 AWS_QUERYSTRING_EXPIRE = 7 * 24 * 60 * 60  # 7 days
 AWS_S3_CUSTOM_DOMAIN = AUTH_TOKENS.get('AWS_S3_CUSTOM_DOMAIN', 'edxuploads.s3.amazonaws.com')
+AWS_S3_ENCRYPTION = True
```

Comment:
Encryption should *not* be enabled by default as it would break the currently deployed instances.
We should leave a backwards compatible value here (`False`), and add this to the storage configuration variables in the configuration playbooks.

Reply:
It makes sense. I'll update the default value.